pages/new/index.tsx

```diff
@@ -61,12 +67,26 @@ function HomePage({ policyType, csrfToken, lang }: HomePageProps) {
 }
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
-  const { csrfToken } = getCsrfToken(context.req, context.res);
+  const tokenFromRequestCookie = parseTokenFromPostRequestCookies(context.req);
   const { query } = context;
-  context.res.setHeader(
-    "Set-Cookie",
-    `nyplUserRegistered=false; Max-Age=-1; path=/; domain=${cookieDomain};`
-  );
+  let newTokenCookie;
```

Comment:
This is the only actual logical update, the rest of the PR is just a refactor with no real functional changes. I don't like how this looks, any suggestions on how to clean it up would be appreciated.